ignite/internal/tools/gen-config-doc/cmd/root.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	v0 "github.com/ignite/cli/v29/ignite/config/chain/v0"
+	v1 "github.com/ignite/cli/v29/ignite/config/chain/v1"
+	"github.com/ignite/cli/v29/ignite/pkg/clidoc"
+	"github.com/ignite/cli/v29/ignite/pkg/cliui"
+	"github.com/ignite/cli/v29/ignite/pkg/errors"
+	"github.com/ignite/cli/v29/ignite/pkg/xgenny"
+
+	"github.com/ignite/cli/ignite/internal/tools/gen-config-doc/templates/doc"
+)
+
+const (
+	flagVersion = "version"
+	flagOutput  = "output"
+
+	defaultDocPath = "docs/docs/08-references"
```

Comment:
I am not sure it is the right file tbh

Reply:
@toschdev, can you check the best place for us to add it? I made the change so we can generate the doc into an example yaml file, but we need to check where we should put